spectrwm.1

```diff
@@ -287,7 +287,7 @@ Some applications have dialogue windows that are too small to be useful.
 This ratio is the screen size to what they will be resized.
 For example, 0.6 is 60% of the physical screen size.
 .It Ic disable_border
-Remove border when bar is disabled and there is only one window on the region.
+Remove border when bar is disabled or there is only one window on the region.
```

Comment:
I guess this should be '`Remove border when there is only one window on the region.`' as enabling/disabling the bar won't influence the display/removal of the border at all anymore.